src/app/modules/main/module.nim

```diff
@@ -395,7 +396,7 @@ method load*[T](
     conf.COMMUNITIESPORTAL_SECTION_ID,
     SectionType.CommunitiesPortal,
     conf.COMMUNITIESPORTAL_SECTION_NAME,
-    amISectionAdmin = false,
+    memberRole = MemberRole.Owner,
```

Comment:
I'm quite surprised to see this here 😄 
I see it's not about your PR, no fixes needed. Just looks quite suspicious